packages/maskbook/src/plugins/Trader/trader/0x/useTrade.ts

```diff
@@ -27,18 +28,21 @@ export function useTrade(
         if (outputAmount === '0' && !isExactIn) return null
         const sellToken = isNative(inputToken.address) ? 'ETH' : inputToken.address
```

Comment:
If a native token is chosen, it should not always use `'ETH'` as the token address. Instead, please test it with `BNB->Token` and `Matic->Token` pairs on BSC and Polygon to find out the correct token address on those networks.

FYI: 

+ https://matcha.xyz/markets/56/0x1af3f329e8be154074d8769d1ffa4ee058b1dbc3/0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee
+ https://matcha.xyz/markets/137/0x8f3cf7ad23cd3cadbd9735aff958023239c6a063/0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee

Reply:
Thanks @guanbinrui added the new function ```setTokenNativeNetwork(networkType)``` for change native token